.gitignore

```diff
@@ -19,6 +19,8 @@
 *.tar.gz
 *.rar
 
+!lib/*.jar
```

Comment:
Generally, we don't commit libraries, but use a build configuration that knows how to collect them.  This is somewhat involved though, so adding it here "just works"